tendrl/ceph_integration/objects/rbd/__init__.py

```diff
@@ -0,0 +1,35 @@
+from tendrl.commons.etcdobj import EtcdObj
+from tendrl.ceph_integration import objects
+
+
+class Rbd(objects.CephIntegrationBaseObject):
+    def __init__(self, name=None, size=None,
+                 pool_id=None, order=None,
+                 block_name_prefix=None,
+                 format=None, features=None,
+                 flags=None,
+                 *args, **kwargs):
+        super(Rbd, self).__init__(*args, **kwargs)
+
+        self.value = 'clusters/%s/Pools/%s/rbds/%s'
+        self.name = name
+        self.size = size
+        self.pool_id = pool_id
+        self.order = order
+        self.block_name_prefix = block_name_prefix
+        self.format = format
+        self.features = features
```

Comment:
We are not using at the moment, but better to keep all the details from ceph cluster in central store as least.